src/components/PersonRow.tsx

```diff
@@ -0,0 +1,56 @@
+import { Person } from '../types';
+import { PersonLink } from '../components/PersonLink';
+import cn from 'classnames';
+
+const NO_PERSON = '-';
+
+type Props = {
+  person: Person;
+  people: Person[];
+  handlePersonClick: (slug: string) => void;
+  selectedPerson: string | null;
+};
+
+export const PersonRow: React.FC<Props> = ({
+  person,
+  people,
+  handlePersonClick,
+  selectedPerson,
+}) => {
+  const { slug, sex, born, died, motherName, fatherName } = person;
+
+  const mother = people.find(p => p.name === motherName);
+  const father = people.find(p => p.name === fatherName);
```

Comment:
makes sense to calculate it once on a higher level, because now you are re-calculating these on each re-render

At least you can use `useMemo` on these values